conjur/cli.py

```diff
@@ -737,7 +739,8 @@ def run_action(resource, args):
         if not is_testing_env:
             loaded_conjurrc = ConjurrcData.load_from_file()
             if not credential_provider.is_exists(loaded_conjurrc.conjur_url):
-                sys.stdout.write("%s\n" % LOGIN_IS_REQUIRED)
+                # pylint: disable=logging-fstring-interpolation
+                sys.stdout.write(f"{LOGIN_IS_REQUIRED}\n")
```

Comment:
if you decide against my [above comment](https://github.com/cyberark/conjur-api-python3/pull/244/commits/a7b0be00a59dac79e94b75d298a223408596489a#r604609192) then let's also extract the second log message from the init_controller.py ("Successfully initialized the Conjur CLI") and lets join that with the above line so it becomes

sys.stdout.write(f"Successfully initialized the Conjur CLI. To start using the CLI, log in to Conjur\n")